Code/Framework/AzToolsFramework/AzToolsFramework/API/ViewportEditorModeTrackerNotificationBus.h

```diff
@@ -52,13 +52,13 @@ namespace AzToolsFramework
         using BusIdType = ViewportEditorModeInfo::IdType;
         //////////////////////////////////////////////////////////////////////////
 
-        //! Notifies subscribers of the a given viewport to the entering of the specified editor mode.
-        virtual void OnEditorModeEnter([[maybe_unused]] const ViewportEditorModesInterface& editorModeState, [[maybe_unused]] ViewportEditorMode mode)
+        //! Notifies subscribers of the a given viewport to the activation of the specified editor mode.
+        virtual void OnEditorModeActivate([[maybe_unused]] const ViewportEditorModesInterface& editorModeState, [[maybe_unused]] ViewportEditorMode mode)
```

Comment:
uber nit: This is is super minor sorry but having this be `OnEditorModeActivated` (past tense) will be useful if we ever want to have a pre/post activate (I assume this is post now?) - If can then be `OnEditorModeActivating` and `OnEditorModeActivated` (same for the deactivate case, it would be `OnEditorModeDeactivated`)

Reply:
fixed 👍

Reply:
Thanks! 😄